addons/zylann.hterrain/tools/plugin.gd

```diff
@@ -852,6 +873,60 @@ func _debug_spawn_collider_indicators():
 				mi.position = hit.position
 
 
+
+func _show_brush_size_selector(min_size, max_size, widget_color, initial_value, action_name, on_value_selected):
+	if _widget_action_name != action_name:
+		var active_viewport = _get_active_viewport()
+
+		if active_viewport == null:
+			# when mouse outside 3d viewport area, dont try to show widget
+			return
+
+		_widget_action_name = action_name
+		if active_viewport != null:
+			var selector_scene = ResourceLoader.load("res://addons/zylann.hterrain/tools/brush/brush_size_selector.tscn")
+			var selector = selector_scene.instantiate()
+
+			selector.min_value = min_size
+			selector.max_value = max_size
+			selector.brush_preview_color = widget_color
+
+			_overlay_selector = selector
+			_overlay_selector.position = active_viewport.get_global_mouse_position()
+			_overlay_selector.set_meta("action", action_name)
+
+			EditorInterface.get_base_control().add_child(_overlay_selector)
+			selector.set_initial_value(initial_value)
+
+			selector.on_value_selected.connect(func f(value):
+				on_value_selected.call(value)
+				_remove_overlay_selector()
+			)
+			selector.on_cancel.connect(func f():
+				_remove_overlay_selector()
+			)
+
+
+func _remove_overlay_selector():
```

Comment:
My bad. I did that, but when refactoring things, I must have missed it.
I had to redo my edits manually later because I had "trim whitespaces on save" in my godot and it messed up the git staging of all files by removing things on every file.